natss/pkg/apis/messaging/v1alpha1/natss_channel_types.go

```diff
@@ -20,9 +20,9 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
-	eventingduck "knative.dev/eventing/pkg/apis/duck/v1alpha1"
+	eventingduck "knative.dev/eventing/pkg/apis/duck/v1beta1"
 	"knative.dev/pkg/apis"
-	duckv1alpha1 "knative.dev/pkg/apis/duck/v1alpha1"
```

Comment:
We should also make sure that the annotation is updated properly?
https://github.com/knative/eventing/blob/master/docs/spec/channel.md

```
- apiVersion: messaging.knative.dev/v1beta1
  kind: YourChannelType
  metadata:
    annotations: messaging.knative.dev/subscribable: v1beta1
```

Reply:
done. Please check it's correctly done, thx!